src/common/dialog/DCFormWrapper.svelte

```diff
@@ -27,43 +33,40 @@
     position: relative;
   }
 
-  fieldset {
-    border: #eee 1px groove;
-  }
 </style>
 
 {#if schema}
   {#if schema.type == "object" || schema.type == "array"}
-    <fieldset>
-      <legend>{schema.title}</legend>
-      <table>
-        <tbody>
-          {#if schema.description}
-            <tr class="field">
-              <td>
-                <div class="description">{schema.description}</div>
-              </td>
-            </tr>
-          {/if}
+    <legend>{schema.title}</legend>
+    {#if schema.description}
```

Comment:
I pulled the description out of the table - the first columns size was growing to accommodate it.  I also removed the fieldset because it is inconsistent with out other forms.